src/config/constants.ts

```diff
@@ -179,7 +180,7 @@ export const AppConfigStatus = {
 
 export const FullRoutes = {
     LOGIN: `${Routes.LOGIN}`,
-    CENTRAL: 'https://api.devtron.ai',
+    CENTRAL: 'https://api-stage.devtron.ai'//'https://api.devtron.ai',
```

Comment:
Change this with prod URL or have some conditions to switch between prod & pre-prod

Reply:
Will do before merge